### DISCUSSION TITLE
Email domain could not be verified

### DISCUSSION THREAD

**Discussion by d-knuth** [community/community#69327] [New to GitHub]
Hi,
I try to use my email *@gmx-topmail.de to sign up.
But i always get the message:

Error adding ***@gmx-topmail.de: Email domain could not be verified.

It doesn't matter what I put before the @. So I guess the "gmx-topmail.de" part of the address is the problem.
Does anybody know how to get the email domain "gmx-topmail.de" verified?

Best regards

**Comment by Muhammad-Abdullah-Jami** [comment]
Dear,
There must be typo error in the email you wrote
i received the similar error and i resolved it by correcting the email.
just review your email it would solve the problem

**Comment by meiser79** [comment]
Hi,

the same happens to me. And there's no typo in my email address.
I'd be very grateful if this issue was fixed.

Thanks,
meiser

**Comment by meiser79** [comment]
Hello,

how can we get some attention on this issue?

Regards,
meiser

**Comment by koeppel** [comment]
I'd suggest contacting the GitHub Support: https://support.github.com/contact/account

**Comment by itshivams** [comment]
### If you are experiencing difficulties signing up on GitHub with the email domain "gmx-topmail.de" and receiving the error message:

> Error adding ***@gmx-topmail.de: Email domain could not be verified.

GitHub has specific procedures for email verification, and certain domains may face issues during this process.

Here are steps to address the problem:

1. **Contact GitHub Support:**
   Reach out to GitHub Support for assistance. Provide them with details about the issue and inquire about the verification process for the "gmx-topmail.de" domain.

2. **Alternative Email:**
   As a temporary solution, consider using an alternative email address with a different, widely-recognized domain to complete the GitHub signup process.

3. **Check GitHub Documentation:**
   Review GitHub's documentation or support resources for any specific instructions or requirements related to email domain verification.

**Hope it helps!**

If the problem persists, contact GitHub Support directly for personalized assistance in resolving the email domain verification issue.

**Comment by meiser79** [comment]
Feedback from Github support:

> I've had a look into this for you, and unfortunately the @gmx-topmail.de domain is one which we have experienced an increased number of issues with, and as such it is no longer possible to verify email addresses from that domain.
>  
> Whilst I know it may be inconvenient, I'm afraid you will need to add another email from a domain which you are able to successfully verify.

**Comment by theRoboxx** [comment]
@simplelogin.com causes the same Error

**Reply by EmilianoJordan** [reply]
This is still happening to me. Unfortunate because a lot of us want to use unique emails per service to strengthen our auth profile.

**Reply by ChrisJBurns** [reply]
Same with `@passmail.net` for Proton Unlimited users. I similar to above want separate emails per service and I can't for Github unless I use a real Proton email and not a "hide-my-email" address.

**Reply by verinikat** [reply]
> Same with `@passmail.net` for Proton Unlimited users.

@nolnah93 points out that `passmail.**com**` will work. It's still working today.

**Comment by kubernetes-bad** [comment]
Emails @horsefucker.org produce the same error, for some reason 🤔

**Comment by dps-910** [comment]
@cock.li emails have the same issue

**Comment by eylenburg** [comment]
And @airmail.cc emails (which is hosted by Cock.li)

Seems like Github doesn't like email hosters that don't require a phone number or otherwise doxxing yourself!

**Comment by cprovatas** [comment]
anonaddy.me doesn't work either

**Comment by alexeyev** [comment]
Same with mail.ee

**Comment by TheJNXx** [comment]
Same here, waifu.club from cock.li

**Comment by fczuardi-luizalabs** [comment]
same with passmail.net

**Reply by nolnah93** [reply]
Fortunately, you can change the alias to passmail.com. it worked for me!

**Comment by HkK2003** [comment]
More or less all cock.li domain are banned.
Same here with memeware.net

**Comment by hoesknowhoes** [comment]
I suspect there is a negative weight for: NRD, poorly configured security DNS RR, and whatever 'reputation' now means

**Comment by kayodde** [comment]
use.startmail.com is banned
But not startmail.com?
Why?
Fix this github!!! I don't want to use my main email god damnit!

**Reply by laniakea64** [reply]
Hi, I too would like to add a `@use.startmail.com` email as a second/additional email address on my Github account, but trying to add it myself in account settings just gives me the "Email domain could not be verified" error.  Was hoping to contact GitHub Support, but the options they now provide for possible reasons for contacting support seem not to include anything that fits inability to add & verify a valid email address?  So asking here since this discussion is the only relevant result that comes up in internet searching about this issue:

How to get help getting my current secondary email address added to my account?

Some responses in this thread indicate this error can happen if Github blocks a domain.  But in the case of use.startmail.com, no idea why it would be blocked?  This domain is how Startmail provides secondary email address to their customers.  Of note, getting a `@use.startmail.com` email address [requires providing valid payment details to Startmail](https://support.startmail.com/hc/en-us/articles/360014652878-Why-your-payment-details-are-needed-for-a-trial-account).

Again, to be clear, in my case I'm **not** looking to change or remove the existing email address currently associated with my GitHub account.  I'd like to leave that one in place and keep it as "primary".  I'd just like to add a secondary email address and use it for notifications.

Thanks for any help - either in directly resolving this, or in how to reach GitHub Support to assist.

**Comment by kellyfelkins** [comment]
mozmail.com also generates this error:

Error adding [redacted]@mozmail.com: Email domain could not be verified. 

mozmail.com is the domain used by the relay service at mozilla - https://relay.firefox.com

**Reply by kellyfelkins** [reply]
I just discovered that the relay project is hosted on github...

https://github.com/mozilla/fx-private-relay

**Reply by kellyfelkins** [reply]
What @EmilianoJordan said:
"This is still happening to me. Unfortunate because a lot of us want to use unique emails per service to strengthen our auth profile."

**Comment by Drake53** [comment]
I tried using a @passinbox.com email but get the same error message. If this is not resolved soon I will delete my account.

**Comment by equinox1179** [comment]
All SimpleLogin domains are banned. Why? It isn't like they are temporary throwaway mails. And even if, what should GitHub care? It's none of their business. If we want to add it as our contact mail, let us do that. 

Hosting the code of several email alias services, and then banning said services' aliases. Such double standards.

**Comment by atorrez007** [comment]
I was able to use a SimpleLogin email. I did need to create a custom subdomain and use that one. Those are catch-all accounts though, so you may not want to use them for GH, but it worked.

**Comment by pmpkn314** [comment]
It's another way for github to collect our data and identify us
